packages/react-data-grid/src/ColumnMetrics.js

```diff
@@ -137,12 +136,12 @@ function compareEachColumn(prevColumns, nextColumns, isSameColumn) {
   return true;
 }
 
-function sameColumns(prevColumns, nextColumns, isSameColumn) {
+export function sameColumns(prevColumns, nextColumns, isSameColumn) {
   if (areColumnsImmutable(prevColumns, nextColumns)) {
     return prevColumns === nextColumns;
   }
 
   return compareEachColumn(prevColumns, nextColumns, isSameColumn);
 }
 
-module.exports = { recalculate, resizeColumn, sameColumn, sameColumns };
+export { sameColumn };
```

Comment:
This is just a re-export, is it necessary to have it here?

Reply:
I need to revisit the entire `ColumnMetrics.js` logic and we can refine it then. Currently we are using `ColumnMetrics.sameColumns` in a few places